### PR TITLE
Adds more info and closes #362

### LINKED ISSUE #362: Add lunchtime talks
To curriculum page

### PR DESCRIPTION
**Pull request by nikeshashar**
Adds information about CTO talks and lunchtime talks to the schedule. Closes #362

### CODE REVIEW

File: source/partials/curriculum/_daily_schedule.html.haml [hunk 1]
```diff
@@ -35,7 +35,7 @@
             %td{data:{header: "Time"}} 1.00pm
             %td{data:{header: "Activity"}} Lunch
             %td{data:{header: "Description"}}
-              %p Our cohorts tend to lunch together. Generally they like to go out in droves to one of Shoreditch’s fine local eateries.
+              %p Our cohorts tend to lunch together. Generally they like to go out in droves to one of Shoreditch’s fine local eateries. There's often a lunchtime talk from one of the many CTO's that our hiring team speak to!
```
Comment (roidriscoll) [inline source/partials/curriculum/_daily_schedule.html.haml]: "local eateries and there's" might flow more easily?

**Comment by roidriscoll** [comment]
I left a copy suggestion, but visually I don't think it will upset the applecart, so have yourself a morally-questionable acorn botherer: :shipit:

File: source/partials/curriculum/_daily_schedule.html.haml (hunk 1 above)
Comment (pitchinvasion) [inline source/partials/curriculum/_daily_schedule.html.haml]: CTOs

**Comment by pitchinvasion** [comment]
No squirrel until that abhorrent apostrophe is removed

**Comment by nikeshashar** [comment]
@pitchinvasion @roidriscoll now?

**Comment by pitchinvasion** [comment]
:shipit:

**Pull request merged**